tools/codegen.mk

```diff
@@ -19,7 +19,7 @@ ifeq (, $(shell which controller-gen))
 		set -e ;\
 		CONTROLLER_GEN_TMP_DIR="$$(mktemp -d)" ;\
 		cd "$$CONTROLLER_GEN_TMP_DIR" ;\
-		go install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.5.0 ; \
+		go install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.11.3 ; \
```

Comment:
will there be any issues increasing the version of controller-get?

Reply:
Shouldn't be an issue. Some files were already compiled with a newer version https://github.com/dapr/dapr/blob/e8d10ae0ae726945b4aee26ac1baca4c86fecb7a/charts/dapr/crds/configuration.yaml#L7

Some whitespaces may be different.